Formalist/AccessoryViewToolbar.swift

```diff
@@ -21,19 +21,24 @@ class AccessoryViewToolbar: UIToolbar {
         return UIBarButtonItem(title: "Next", style: .plain, target: self, action: #selector(nextAction))
     }()
 
-    lazy var doneButtonItem: UIBarButtonItem = {
-        return UIBarButtonItem(barButtonSystemItem: .done, target: self, action: #selector(done))
-    }()
+    private func doneButtonItem(customTitled title:String? = nil) -> UIBarButtonItem {
+        if let title = title {
+            return UIBarButtonItem(title: title, style: .plain, target: self, action: #selector(done))
```

Comment:
Did you consider adding `style` as well for consistency, in that case it would be consistent, since there is one `.plain` and `.done`

Reply:
Do you mean to expose the `DoneButtonStyle` to both the `TextEditorConfiguration` and the `ActivityViewToolbar`?